### DISCUSSION TITLE
GitHub Student Benefits Lost

### DISCUSSION THREAD

**Discussion by zihanghliu** [community/community#83239] [GitHub Education]
### Select Topic Area

Bug

### Body

Hi there. My account used to have student benefits of pro features like GitHub copilot, and I have verified my student email and joined the global campus. But recently my account was flagged, and after my account was reinstated, somehow all my pro features were gone, and when I try to reactivate the student benefit by rejoining the GitHub global campus, in the page [Student Verification](https://education.github.com/pack/join) when I click the button to confirm I am a student, it went to a [Pricing Page](https://github.com/pricing) which I believe it's not normal and I cannot proceed to the student verification in any way. Can someone please tell me what the problem is and how to get my student status and pro features back?

**Comment by abhishekHegde2000** [comment]
hi can you tell me how were you able to  reinstate your account??? my account is showing its flagged also

**Reply by medjamac** [reply]
Ok

**Comment by abhishekHegde2000** [comment]
Hey can you please tell me how many days does it take to reinstate the account?? its already been 4 days for me but i did not get any replies from Support

**Comment by medjamac** [comment]
Pokmm

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HenryLiu0820. We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.

I've gone ahead and moved this to the correct category for you. Good luck!
